webapp/components/PostCard/index.vue

```diff
@@ -1,69 +1,71 @@
 <template>
-  <ds-flex-item :width="width">
-    <ds-card
-      :image="post.image | proxyApiUrl"
-      :class="{ 'post-card': true, 'disabled-content': post.disabled }"
+  <ds-card
+    :image="post.image | proxyApiUrl"
+    :class="{ 'post-card': true, 'disabled-content': post.disabled }"
+  >
+    <!-- Post Link Target -->
+    <nuxt-link
+      class="post-link"
+      :to="{ name: 'post-id-slug', params: { id: post.id, slug: post.slug } }"
     >
-      <!-- Post Link Target -->
-      <nuxt-link
-        class="post-link"
-        :to="{ name: 'post-id-slug', params: { id: post.id, slug: post.slug } }"
-      >
-        {{ post.title }}
-      </nuxt-link>
-      <ds-space margin-bottom="small" />
-      <!-- Username, Image & Date of Post -->
-      <div>
-        <no-ssr>
-          <hc-user :user="post.author" :trunc="35" :date-time="post.createdAt" />
-        </no-ssr>
-        <hc-ribbon :text="$t('post.name')" />
+      {{ post.title }}
+    </nuxt-link>
+    <ds-space margin-bottom="small" />
+    <!-- Username, Image & Date of Post -->
+    <div>
+      <no-ssr>
+        <hc-user :user="post.author" :trunc="35" :date-time="post.createdAt" />
+      </no-ssr>
+      <hc-ribbon :text="$t('post.name')" />
+    </div>
+    <ds-space margin-bottom="small" />
+    <!-- Post Title -->
+    <ds-heading tag="h3" no-margin>{{ post.title }}</ds-heading>
+    <ds-space margin-bottom="small" />
+    <!-- Post Content Excerpt -->
+    <!-- eslint-disable vue/no-v-html -->
+    <!-- TODO: replace editor content with tiptap render view -->
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T22:35:24Z" title="Friday, August 16th 2019, 12:35:24 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -123,6 +132,26 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    calculateItemHeight() {
```


@alina-beck do you still need this? The build server is crashing during frontend testing and it's trying to set some `style` attribute of `null`.